package.json

```diff
@@ -115,6 +115,7 @@
     "http-server": "^0.11.1",
     "husky": "^0.14.3",
     "jest": "^23.5.0",
+    "jest-environment-jsdom-thirteen": "^0.0.2",
```

Comment:
The root cause is that jest aims to support node v4 (bumped to node v6 for latest version of jest 24), while latest versions of jsdom supports only node v8. So Jest doesn't want to upgrade their jsdom dependency...

I think this is useful for us to ugprade because this is closer to the browser environment.